app/extensions.py

```diff
@@ -1,6 +1,7 @@
 import pyrebase
 import os
 from urllib.parse import quote
+import json
```

Comment:
Python Linter caught this: `./app/extensions.py:4:1: F401 'json' imported but unused`

Reply:
Will remove line, I don't remember using json for this file but it might have slipped on those late night grinds haha